.eslintrc.json

```
@@ -0,0 +1,15 @@
+{
+  "env": {
+    "browser": true,
```

Shouldn't this be:

```suggestion
    "node": true,
```

I'm not entirely sure, this just caught my eye.